src/common/marlin_client.c

```diff
@@ -140,6 +130,7 @@ void marlin_client_loop(void) {
             if (client->flags & MARLIN_CFLG_LOWHIGH) {
                 *(((uint32_t *)(&msg)) + 1) = ose.value.v; //store high dword
                 _process_client_message(client, msg);      //call handler
+                variant8_done(&msg);
```

Comment:
are you 100% sure the pointer held in `msg` is to be dealocated here calling `variant8_done`?

Reply:
yes, this is the right place - immediately after message processed.

Reply:
Resource is allocated in function marlin_vars_get_var, variant8_done does nothing in case of nonpointer type or empty data. Each client will receive own copy.

Reply:
ok, in this case the code path seems to be deterministic